internal/cli/capact/components.go

```diff
@@ -376,23 +399,23 @@ func (k *Kubed) InstallUpgrade(version string) (*release.Release, error) {
 func (c *Capact) InstallUpgrade(version string) (*release.Release, error) {
 	upgradeCli := c.upgradeAction(version)
 
-	capactValues := c.opts.Parameters.Override.CapactValues
-	if version == LocalVersionTag {
-		capactValues.Global.ContainerRegistry.Path = LocalDockerPath
-		capactValues.Global.ContainerRegistry.Tag = LocalDockerTag
-	}
-	s := structs.New(capactValues)
-	s.TagName = "json"
-	mappedValues := s.Map()
+	capactValues := c.opts.Parameters.Override.CapactValues.AsMap()
 
 	if c.opts.Environment == KindEnv {
 		values, err := ValuesFromString(capactKindOverridesYaml)
 		if err != nil {
 			return nil, errors.Wrap(err, "while converting override values")
 		}
-		mappedValues = tools.MergeMaps(values, mappedValues)
+		capactValues = tools.MergeMaps(values, capactValues)
 	}
-	return c.runUpgrade(upgradeCli, mappedValues)
+
+	for _, value := range c.opts.Parameters.Override.CapactStringOverrides {
```

Comment:
it's done already in `SetCapactValuesFromOverrides` do we need to repeat that?

so maybe we should get rid of `SetCapactValuesFromOverrides` or unify our approach and have same func for `cert-manager` and `ingress`

Reply:
In SetCapactValuesFromOverrides we only set values in the struct CapactValues. Other values are ignored there. Some of these values are used before installing Capact.


Here we pass all values.